src/gui/aboutdialog.cpp

```diff
@@ -113,8 +112,9 @@ AboutDialog::AboutDialog(QWidget *parent)
                                      "The database is licensed under the Creative Commons Attribution 4.0 International License"));
     m_ui->labelDBIP->setText(DBIPText);
 
-    resize(m_storeDialogSize);
-    show();
+    const QSize dialogSize = m_storeDialogSize;
```

Comment:
Maybe move it in initialization of `if`? E.g.:
```suggestion
if(const QSize dialogSize = m_storeDialogSize; dialogSize.invalid())
```

Reply:
Done.